services/121-service/src/cronjob/cronjob.controller.ts

```diff
@@ -21,7 +21,7 @@ export class CronjobController {
     type: [RunCronjobsResponseDto],
   })
   public async runAllCronjobs(): Promise<RunCronjobsResponseDto[]> {
-    const methods = this.cronjobService.getAllMethods().methodNames;
+    const methods = this.cronjobService.getAllMethods();
```

Comment:
```suggestion
    const methods = this.cronjobService.getAllMethodNames();
```

Reply:
Ohow. This GitHub-UI makes it way too easy to "quickly fix a typo"...
But actually, IDEs are way better at that. :D